### ISSUE TITLE
"Missing Local Resources Checker" needs reference counter 

### ISSUE THREAD

**Issue by gernotstarke** [aim42/htmlSanityCheck#42]
the same missing resource is reported multiple times, e.g.:

```
local resource "api/griffon/core/env/Metadata.html" missing
local resource "api/griffon/core/env/Metadata.html" missing
local resource "api/griffon/core/env/Feature.html" missing
local resource "api/griffon/core/env/Feature.html" missing
local resource "api/griffon/core/env/Metadata.html" missing
local resource "api/griffon/core/env/Metadata.html" missing
```

Should better be:
local resource "api/griffon/core/env/Metadata.html" missing (reference count: 4)
local resource "api/griffon/core/env/Feature.html" missing (reference count: 2)
